<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplyInputWidget</class>
 <widget class="QWidget" name="ReplyInputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>631</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTextEdit" name="txt_question">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="bnt_icon_question">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="txt_reply">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="bnt_icon_reply">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="bnt_cancel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>انصراف</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="bnt_ok">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>ثبت پاسخ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
